specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -12806,6 +15566,90 @@
         }
       }
     },
+    "ContinuousWebJob": {
+      "description": "Continuous Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ContinuousWebJob resource specific properties",
+          "properties": {
+            "status": {
+              "description": "Job status.",
+              "type": "string"
+            },
+            "detailedStatus": {
+              "description": "Detailed status.",
+              "type": "string"
+            },
+            "logUrl": {
+              "description": "Log URL.",
+              "type": "string"
+            },
+            "name": {
```

Comment:
Why is this property needed again? Name of the resource comes from the URL and is also present in the ARM top level name property

Reply:
I still see this here: https://github.com/EricSten-MSFT/azure-rest-api-specs/blob/449478bf179eae0001b9fd463355dd517e74fb6a/specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json#L16852

Reply:
I'm looping back with the owning devs to figure out why this is here.  I don't think I can ignore the property, since Kudu may be depending upon it.

Reply:
A collection of ContinuousWebJobs is returned from ListContinuousWebJobs.  The "name" field is needed to differentiate between the web jobs in the collection.

Reply:
Confirmed with owning devs: the name field is read-only and not used when creating.

Reply:
fix should be in Update #5